runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -209,7 +209,11 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
             OMR::CriticalSection getAddressSetRanges(assumptionTableMutex);
             unloadedClasses->getRanges(ranges);
             }
-         client->write(response, ranges, unloadedClasses->getMaxRanges());
+         // Add the entire CHTable as well
+         auto table = (JITClientPersistentCHTable*)comp->getPersistentInfo()->getPersistentCHTable();
+         std::string encoded = FlatPersistentClassInfo::serializeHierarchy(table);
+
+         client->write(response, ranges, unloadedClasses->getMaxRanges(), encoded);
```

Comment:
Can we change the message name to `getUnloadedClassRangesAndCHTable` or something like that? It would make the rest of the code a little clearer.

Reply:
Done